automation/taskcluster/decisionlib.py

```diff
@@ -240,6 +236,9 @@ def find_or_create(self, index_path=None):
 
         <https://docs.taskcluster.net/docs/reference/core/taskcluster-index/references/api#findTask>
         """
+        # In release environments we always create a task, which ensures chainOfTrust is set.
```

Comment:
Rather than push this flag down to `decisionlib`, accept more parameters in `with_dockerfile` -- say, one that will _never_ use the index and one for setting the index prefix.